documentation/modules/operators/proc-configuring-multiple-cluster-operators.adoc

```diff
@@ -0,0 +1,122 @@
+Use of Role Base Access Control (RBAC) in the Kubernetes cluster usually means that permission to create, edit, and delete these resources is limited to Kubernetes cluster administrators, such as `system:admin`.
+
+.Procedure
+
+Edit the `Deployment` resource that is used to deploy the Cluster Operator, which is defined in the `060-Deployment-strimzi-cluster-operator.yaml` file.
+
+. Change the `replicas` property from the default (1) to a value that matches the required number of replicas.
++
+[source,yaml,numbered,options="nowrap",highlight='12']
+.Increasing the number of Cluster Operator replicas
+----
+include::../install/cluster-operator/060-Deployment-strimzi-cluster-operator.yaml[lines=1..7] 
+  replicas: 3 
+----
+
+. Configure leader election `env` properties.
++
+To enable leader election, `STRIMZI_LEADER_ELECTION_ENABLED` must be set to `true` (default).
++
+In this example, the name of the lease is changed to `my-strimzi-cluster-operator`.
++
+[source,yaml,numbered,options="nowrap",highlight='12']
+.Configuring leader election environment variables for the Cluster Operator
+----
+# ... 
+spec
+  containers:
+    - name: strimzi-cluster-operator
+      # ...
+      env:
+        - name: STRIMZI_LEADER_ELECTION_ENABLED
+          value: "true"
+        - name: STRIMZI_LEADER_ELECTION_LEASE_NAME
+          value: "my-strimzi-cluster-operator"
+        - name: STRIMZI_LEADER_ELECTION_LEASE_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+        - name: STRIMZI_LEADER_ELECTION_IDENTITY
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.name
+----
++
+For a description of the available environment variables, see xref:ref-operator-cluster-leader-election-{context}[].
++
+If you specified a different name or namespace for the `Lease` resource used in leader election, update the RBAC resources.
+
+. (optional) Edit the `ClusterRole` resource in the `022-ClusterRole-strimzi-cluster-operator-role.yaml` file.
++
+Update `resourceNames` with the name of the `Lease` resource.
++ 
+[source,yaml,numbered,options="nowrap",highlight='12']
+.Updating the ClusterRole references to the lease 
+----
+include::../install/cluster-operator/022-ClusterRole-strimzi-cluster-operator-role.yaml[lines=1..7;21]
+      - my-strimzi-cluster-operator
+# ...      
+----
+
+. (optional) Edit the the `RoleBinding` resource in the `022-RoleBinding-strimzi-cluster-operator.yaml` file.
++
+Update `subjects.name` and `subjects.namespace` with the name of the `Lease` resource and the namespace where it was created.
++ 
+[source,yaml,numbered,options="nowrap",highlight='12']
+.Updating the RoleBinding references to the lease 
+----
+include::../install/cluster-operator/022-RoleBinding-strimzi-cluster-operator.yaml[lines=1..8]
+    name: my-strimzi-cluster-operator
+    namespace: myproject
+# ...
+----
+
```

Comment:
Since this is all done by default ... should we formulate it more as _Doublecheck that these are set and if not set them?

Reply:
Okay. I changed to:

> . Check that the leader election `env` properties are set.
> +
> If they are not set, configure them.